lisa/microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -395,17 +419,46 @@ def _install_driver(node: Node, log_path: Path, log: Logger) -> None:
         reboot_tool.reboot_and_check_panic(log_path)
         return
     except UnsupportedOperationException:
-        log.info("Installing NVIDIA Driver using Azure GPU Extension is not supported")
+        log.info("Installing Driver using Azure GPU Extension is not supported")
     except Exception:
-        log.info("Failed to install NVIDIA Driver using Azure GPU Extension")
+        log.info("Failed to install Driver using Azure GPU Extension")
         if isinstance(node.os, Ubuntu):
             # Cleanup required because extension might add sources
             sources_after = node.execute(
                 "ls -A1 /etc/apt/sources.list.d", sudo=True
             ).stdout.split("\n")
             __remove_sources_added_by_extension(node, sources_before, sources_after)
 
-    __install_driver_using_sdk(node, log, log_path)
+    # Install LIS driver if required (for older kernels)
+    try:
```

Comment:
@LiliDeng maybe we can remove it totally? It's in CentOS 6.x and may not be supported by modern GPUs.

Reply:
The comment is still relevant, I just moved the code to a different method